### PR TITLE
Bump PonyDebugger to use the lastest SocketRocket

### PR DESCRIPTION
**Pull request by holmes**
I don't really know what I'm doing here.

### CODE REVIEW

File: ObjC/PonyDebugger.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -846,7 +865,7 @@
 					"\"$(CONFIGURATION_BUILD_DIR)/usr/local/include\"",
 					./SocketRocket,
 				);
-				IPHONEOS_DEPLOYMENT_TARGET = 5.0;
+				IPHONEOS_DEPLOYMENT_TARGET = 9.1;
```
Comment (justinseanmartin) [inline ObjC/PonyDebugger.xcodeproj/project.pbxproj]: Internally at least, we still build with 9.0 support. Is there a reason to drop support for all of these versions? We should at least support 9.0, if not 8.0 as well.

**Comment by justinseanmartin** [comment]
We should also bump the version of SocketRocket consumed by the podspec. The SocketRocket submodule reference bump will only work for consumers pulling in PonyDebugger as a submodule.

**Comment by justinseanmartin** [comment]
Looks like 0.5.1 is the latest. We should probably ensure that the version pulled from cocoapods and the one that the submodule reference SHA are the same as well.

**Comment by justinseanmartin** [comment]
Also do a PonyDebugger.podspec version bump

File: ObjC/PonyDebugger.xcodeproj/project.pbxproj (hunk 1 above)
Comment (holmes) [inline ObjC/PonyDebugger.xcodeproj/project.pbxproj]: That was the lowest version the drop down let me pick. I'll change to 8.0.

**Comment by justinseanmartin** [comment]
I take back my statement on CocoaPods, the podspec isn't restrictive to version of SocketRocket. LGTM.

**Comment by holmes** [comment]
Waiting on a new version of SocketRocket, see https://github.com/facebook/SocketRocket/issues/513

File: PonyDebugger.podspec [hunk 2]
```diff
@@ -9,9 +9,9 @@ Pod::Spec.new do |s|
   s.license         =  'Apache License, Version 2.0'
 
   s.requires_arc = true
-  s.ios.deployment_target = '5.0'
+  s.ios.deployment_target = '8.0'
   s.source_files = 'ObjC/{DerivedSources,PonyDebugger}/**/*.{h,m}'
   s.frameworks = 'CoreData', 'CoreGraphics'
-  s.dependency 'SocketRocket'
+  s.dependency 'SocketRocket' '0.5.1'
```
Comment (asmallteapot) [inline PonyDebugger.podspec]: Please add a comma between the dependency name and version, like this:

```ruby
s.dependency 'SocketRocket', '0.5.1'
```

This is due to a quirk in [Ruby's string literal syntax](http://ruby-doc.org/core-2.5.1/doc/syntax/literals_rdoc.html#label-Strings):

> Adjacent string literals are automatically concatenated by the interpreter:
>
> ```ruby
> "con" "cat" "en" "at" "ion" #=> "concatenation"
> "This string contains "\
> "no newlines."              #=> "This string contains no newlines."
> ```

**Comment by mthole** [comment]
I think we can close this out. As part of https://github.com/square/PonyDebugger/pull/212, I updated to SocketRocket 0.5.1. That's the latest "release" available (see https://github.com/facebook/SocketRocket/releases).

**Pull request closed without merging**